### PR TITLE
Fixed #4601 Link missing on page: /guides/v2.3/frontend-dev-guide/translations/xlate.html and v2.2

### LINKED ISSUE #4601: Link missing on page: /guides/v2.3/frontend-dev-guide/translations/xlate.html
# General issue

There is a placeholder / missing link on the Translation page.

## Description:

Under [the heading](https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/xlate.html#m2devgde-xlate-generatortool) this link does not exist:
> For more information, see [dictionary generator tool].

## Possible solutions:

If there is currently a page on this it should be included. If not, perhaps change the sentence to: "We intend to publish more information on this technique at a later time."

### PR DESCRIPTION
**Pull request by kirtinariya1**
## Purpose of this pull request

This pull request (PR) will be merge the show correct in the Dictionary generator tool content

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/translations/xlate.html#m2devgde-xlate-generatortool
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/translations/xlate.html#m2devgde-xlate-generatortool

## Links to Magento source code
N/A

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

**Comment by dobooth** [comment]
Hi @kirtinariya1 Thanks for the contribution. Can you please update your branch and fix this merge conflict. I want to make sure you still credit for the PR.

**Comment by kirtinariya1** [comment]
@dobooth Conflict has resolved.

**Comment by dobooth** [comment]
Thanks @kirtinariya1 I am still seeing a conflict for the v2.3 xlate.md file.

File: guides/v2.2/frontend-dev-guide/translations/xlate.md [hunk 1]
```diff
@@ -105,49 +105,45 @@ Assembling the preceding `pt_BR.csv` files across all modules and the current th
 
 ### Dictionary generator tool {#m2devgde-xlate-generatortool}
 
-You can generate a translation dictionary to use by itself (for example, to translate words and phrases in a custom module) or for use by a [language package](https://glossary.magento.com/language-package). For more information, see [dictionary generator tool].
+You can generate a translation dictionary to use by itself (for example, to translate words and phrases in a custom module) or for use by a [language package](https://glossary.magento.com/language-package). For more information, see [dictionary generator tool - We intend to publish more information on this technique at a later time].
```
Comment (hguthrie) [inline guides/v2.2/frontend-dev-guide/translations/xlate.md]: We do not promise things in the future. This looks like it was supposed to be a link. Possibly the one listed at the bottom of the page `[More information about the dictionary generator tool]: {{page.baseurl}}/config-guide/cli/config-cli-subcommands-i18n.html#config-cli-subcommands-xlate-dict`

Comment (hguthrie) [inline guides/v2.2/frontend-dev-guide/translations/xlate.md]:
```suggestion
You can generate a translation dictionary to use by itself (for example, to translate words and phrases in a custom module) or for use by a [language package](https://glossary.magento.com/language-package). See [dictionary generator tool].
```

Comment (kirtinariya1) [reply guides/v2.2/frontend-dev-guide/translations/xlate.md]: @hguthrie Changes has done.

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=4998) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=4998) before we can accept your contribution.<br/>**2** out of **3** committers have signed the CLA.<br/><br/>:white_check_mark: kirtinariya1<br/>:white_check_mark: dobooth<br/>:x: Kirti Nariya<hr/>**Kirti Nariya** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by dobooth** [comment]
Thanks @kirtinariya1

**Comment by dobooth** [comment]
running tests

**Pull request merged**